supervisor/homeassistant/module.py

```diff
@@ -66,6 +66,7 @@
     "*.log",
     "*.log.*",
     "OZW_Log.txt",
+    "tts/*",
```

Comment:
Let's merge for now, but I think we need to make a task to get this folder out of there...

As in, feels odd to have caching stuff in the user space.